it-and-security/default.yml

```
@@ -31,6 +31,9 @@ org_settings:
   host_expiry_settings:
     host_expiry_enabled: false
   integrations:
+    google_calendar:
+  - api_key_json: $DOGFOOD_CALENDAR_API_KEY
+    domain: fleetdm.com
```

```suggestion
    - api_key_json: $DOGFOOD_CALENDAR_API_KEY
      domain: fleetdm.com
```

Looking at Victor's video, I think we want to tab these in (one level under `google_calendar`)

---

Going to commit this change and approve.